pkg/apis/options/providers.go

```diff
@@ -219,6 +219,9 @@ type OIDCOptions struct {
 	// JwksURL is the OpenID Connect JWKS URL
 	// eg: https://www.googleapis.com/oauth2/v3/certs
 	JwksURL string `json:"jwksURL,omitempty"`
+	// PublicKeys is a list of paths pointing to public key files in PEM format to use
+	// for verifying JWT tokens
+	PublicKeys []string `json:"publicKeys,omitempty"`
```

Comment:
Do you think it should be `PublicKeyFiles`? To avoid confusion with trying to inline the public keys?